### PR TITLE
Remove ORG and add full author name, Part of issue #55

### PR DESCRIPTION
**Pull request by Hang-Lei-NOAA**
This is part of issue #55 
According to Edward's decision, the ORG info is removed.
The Full name of author is added for all doxygen added code files.

### CODE REVIEW

File: src/enc_jpeg2000.c [hunk 1]
```diff
@@ -56,14 +56,14 @@
  * 1 = try increasing number of guard bits otherwise, no additional options
  * @param[in] jpclen  - Number of bytes allocated for new JPEG2000 code stream in outjpc.
  * @param[in] outjpc - Output encoded JPEG2000 code stream
- * @return - > 0 = Length in bytes of encoded JPEG2000 code stream
+ * @return 0 = Length in bytes of encoded JPEG2000 code stream
```
Comment (edwardhartnett) [inline src/enc_jpeg2000.c]: You can remove the = as well, so just:

`@return 0  Length in bytes of encoded JPEG2000 code stream
`

But is this correct? If it returns 0, how is that the length of the stream?

**Review by edwardhartnett — CHANGES_REQUESTED** [review]

File: src/enc_jpeg2000.c (hunk 1 above)
Comment (Hang-Lei-NOAA) [reply src/enc_jpeg2000.c]: Will remove that "=".
The return code just show that the code works good. This will be rewords.

**Review by edwardhartnett — APPROVED** [review]

**Comment by edwardhartnett** [comment]
Not sure why this is closing the issue.

Next time try exactly:

"Part of #N"

Maybe that will work without closing the issue.

**Comment by Hang-Lei-NOAA** [comment]
Okay, Thanks.

**Pull request merged**